### PR TITLE
fix(update data)

### LINKED ISSUE #562: UI don't refresh automatically across the website
- [x] Review and define problem
- [x] Define next steps

Example: As a Lifebank, when transferring tokens to a donor, the UI, don't update automatically to allow a next transaction, it obliges the user to refresh the page manually

Steps to test:
1. Sign in as a Lifebank
2. Go to transfer tokens section
3. Make a transaction (It will throw a message of successful or error) and this don't hide automatically

image

### PR DESCRIPTION
**Pull request by Edgar097**
Resolves #562 

### What does this PR do?
Fix of the donor sign up error
Load offer data after the user is informed of the changes

### How should this be tested?
- Make run
- Add a new donor
- Add a new offer

### CODE REVIEW

**Comment by leisterfrancisco** [comment]
@Edgar097 what does this PR do?

**Review by steph222 — APPROVED** [review]

File: webapp/src/routes/Profile/ProfilePageLifebank.js [hunk 1]
```diff
@@ -606,7 +606,11 @@ const ProfilePageLifebank = ({ profile, loading }) => {
 
 ProfilePageLifebank.propTypes = {
   profile: PropTypes.object,
+<<<<<<< HEAD
```
Comment (JustinCast) [inline webapp/src/routes/Profile/ProfilePageLifebank.js:609]: Hey @Edgar097 take a look at this

**Review by JustinCast — CHANGES_REQUESTED** [review]

**Comment by Edgar097** [comment]
The offer loading data issue is solved in this PR, however the problem in the edition of the profiles still present, we hope we can solve this problem in future issues

**Review by JustinCast — APPROVED** [review]

**Comment by Edgar097** [comment]
Last commit, fix the donations

File: webapp/src/components/Signup/Signup.js [hunk 2]
```diff
@@ -420,7 +420,9 @@ const Signup = ({ isHome, isModal, isSideBar }) => {
 
 
   useEffect(() => {
-    if (errorcreateAccount) setErrorMessage(t('errors.authError'))
+    if (errorcreateAccount) {
+      setErrorMessage(t('errors.authError'))
```
Comment (JustinCast) [inline webapp/src/components/Signup/Signup.js:424]: Why you added these curly braces?

**Review by leisterfrancisco — APPROVED** [review]

**Pull request merged**